src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.PasswordValidationException;
+
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password == null || !password.equals(repeatPassword) || password.length() < 10) {
+            throw new PasswordValidationException("Wrong passwords or length less than 10");
+        } else {
```

Comment:
do we need `else` after `throw`?

Reply:
solved, return without else, because if block "if" == false we don't use them and just return true :)